TrustWalletCore.podspec

```diff
@@ -132,13 +132,15 @@ Pod::Spec.new do |s|
     "#{protobuf_dir}/src/google/protobuf/port_def.inc",
     "#{protobuf_dir}/src/google/protobuf/port_undef.inc",
   s.exclude_files =
-    'trezor-crypto/src/rand.c'
+    'trezor-crypto/src/rand.c',
+    'trezor-crypto/include/TrezorCrypto/**/base58.h'
   s.public_header_files =
```

Comment:
any reason for `base58` exclude?

Reply:
This tries to fix the warnings when you build wallet core from source (pod -> master), we don't use c base58.h in the app